src/Runner.Worker/ActionManifestManager.cs

```diff
@@ -332,10 +336,19 @@ private TemplateContext CreateTemplateContext(
                 }
             }
 
-            // Expression functions from execution context
-            foreach (var item in executionContext.ExpressionFunctions)
+            // Expression functions
+            foreach (var func in executionContext.ExpressionFunctions)
             {
-                result.ExpressionFunctions.Add(item);
+                GitHub.Actions.Expressions.IFunctionInfo newFunc = func.Name switch
+                {
+                    "always" => new GitHub.Actions.Expressions.FunctionInfo<Expressions.NewAlwaysFunction>(func.Name, func.MinParameters, func.MaxParameters),
```

Comment:
These new functions were defined with the last PR:
- https://github.com/actions/runner/pull/4092